src/session_impl.cpp

```diff
@@ -1838,7 +1838,12 @@ namespace aux {
 		{
 			std::string const& device = m_listen_interfaces[i].device;
 			int const port = m_listen_interfaces[i].port;
-			bool const ssl = m_listen_interfaces[i].ssl;
+			bool const ssl =
+#ifdef TORRENT_USE_OPENSSL
```

Comment:
I'm not sure about silently ignoring the request to listen to an SSL socket and still open a socket but _without_ SSL? That could even be a security risk. What do you think about the following instead:

1. post a listen_alert_error with operation_not_supported error
2. skip this port, through "continue".